core/src/main/scala/org/apache/spark/sql/hive/SnappyStoreHiveCatalog.scala

```diff
@@ -626,6 +671,32 @@ class SnappyStoreHiveCatalog(context: SnappyContext)
     }
   }
 
+  class SnappyIndexHandler extends TableBasedIndexHandler {
+    override def usesIndexTable() : Boolean = false
+
+    @throws(classOf[HiveException])
+    override def analyzeIndexDefinition(table: Table, index: Index, table1: Table) : Unit = { }
+
+    def generateIndexQuery(var1: List[Index], var2: ExprNodeDesc,
+                           var3: ParseContext, var4: HiveIndexQueryContext): Unit = {}
+
+  }
+
+  def createIndex(tableIdent: QualifiedTableName, indexName: String,
+                  indexedCols: List[String],
+                  idxProps: java.util.Map[String, String]): Unit = {
```

Comment:
Btw, I have fixed this in the new parser itself where the result of "identifier" non-terminal is always properly cased (as per conf setting, and whether identifier was quoted or unquoted because for former it should always be taken as is). However, still need to get the new parser done for the DDLParser. I am waiting for AQP to finish merge first else my changes will again increase their work and merge will again get stalled. In the meantime, we have to take care of that explicitly using catalog.processIdentifier even though it is not proper to do that for quoted identifiers as I mentioned.